debian/build.go

```diff
@@ -291,6 +292,25 @@ func getKubeadmKubeletConfigFile(v version) (string, error) {
 	}
 }
 
+// CNI get bumped in 1.9, which is incompatible for kubelet<1.9.
+// So we need to restrict the CNI version when install kubelet.
+func getKubeletCNIVersion(v version) (string, error) {
+	sv, err := semver.Make(v.Version)
+	if err != nil {
+		return "", err
+	}
+
+	v190, err := semver.Make("1.9.0")
+	if err != nil {
+		return "", err
+	}
+
+	if sv.GTE(v190) {
+		return fmt.Sprintf(">= %s", cniVersion), nil
+	}
+	return fmt.Sprintf("< %s", cniVersion), nil
```

Comment:
Should we pin to exactly 0.5.1 for k8s <v1.9? We probably don't want this to move the next time we bump `cniVersion`.

Reply:
I think we'd better not pin to a fixed version. `0.5.2` is also available.